webui/src/pages/auth/groups/index.tsx

```diff
@@ -93,6 +94,8 @@ const GroupsContainer = () => {
 
     if (error) return <Error error={error}/>;
     if (loading) return <Loading/>;
+    const {RBAC: rbac} = useLoginConfigContext();
```

Comment:
I've been told that _real_ frontend developers frown upon destructuring the return value from a hook.  So
```suggestion
    const lc = useLoginConfigContext();
```
and use `lc.RBAC` where needed.